test/gateway/ost_prime/constructor.js

```diff
@@ -0,0 +1,81 @@
+const OSTPrime = artifacts.require("OSTPrime")
+  , BN = require('bn.js')
+  , web3 = require('../../test_lib/web3.js');
+
+const Utils = require('../../../test/test_lib/utils');
+
+const NullAddress = "0x0000000000000000000000000000000000000000";
+contract('OSTPrime.sol', function (accounts) {
```

Comment:
We use contract name followed by the method being tested.
Example :-
Contract('OSTPrime.constructor()',function (accounts) {
});